salt/edx/prod.sls

```diff
@@ -159,6 +167,26 @@ run_ansible:
       - virtualenv: create_ansible_virtualenv
     - unless: {{ salt.pillar.get('edx:skip_ansible', False) }}
 
+create_course_asset_symlink:
+  file.absent:
+    - name: /edx/var/edxapp/course_static/
+  file.symlink:
+    - name: /edx/var/edxapp/course_static
+    - target: /mnt/data/repos
+    - makedirs: True
+    - user: edxapp
+    - group: edxapp
+
+create_course_prod_asset_symlink:
+  file.absent:
+    - name: /edx/var/edxapp/prod_course_static/
+  file.symlink:
+    - name: /edx/var/edxapp/prod_course_static
+    - target: /mnt/data/prod_repos
+    - makedirs: True
+    - user: edxapp
+    - group: edxapp
```

Comment:
We can get away with having one symlink. The only reason we had two in our old stack was because the hosts in our old stack serving static assets were the load balancers, which serve both "draft" and "live".

Maybe just set the target to `salt.pillar.get('edx:edxapp:GIT_REPO_DIR')` and use the name `/edx/var/edxapp/course_static/` both here and in the template.